### DISCUSSION TITLE
В таблице циклов, в поле creator_id имя пользователя встает только в зачеркнутом виде

### DISCUSSION THREAD

**Discussion by aityomin** [totumonline/totum-ru-issues-and-discussions#434] [Как это сделать?] [answered]
В секции Код поля creator_id
=: listCreate(item: $user)
user: select(table: 'users'; field: 'login'; where: 'id' = #h_uid;)

В поле h_uid инсертом передается id пользователя.
Пробовал вместе с  id  передавать и имя, но эффект был таким же.

Что я делаю не так?

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Там id:

```
=: selectList(table: 'users'; field: 'id'; where: 'id' = #h_uid)
```

Да как-бы можно и 

```
=: json`[#h_uid]`
```

Меня беспокоит, что используется #h_uid тк это поле в БД и если там 10 пользователей будут тыкать что-то одновременно наступит хаос.

Обычно для всяких селекторов используются поля в Префильтре тк Префильтр индивидуальный для каждого открывшего таблицу прользователя.

**Reply by aityomin** [reply]
Нет. У пользователей будет доступ только к таблицам внутри цикла. А таблицу цикла они по идее даже видеть не должны. То есть каждый новый зарегистрировавшийся пользователь "создает" строку в таблице цикла и соответственно получает таблицу внутри цикла, в качестве личного кабинета.

**Reply by aityomin** [reply]
=: selectList(table: 'users'; field: 'id'; where: 'id' = #h_uid)
Все получилось! Спасибо!

**Reply by ttm-alexeypolunin** [reply]
Плохо делаете — для этой задачи: https://github.com/totumonline/totum-ru-issues-and-discussions/discussions/435

Если делать этим путем то пользователя надо считать не в хедере а в Префильтре!
